### ISSUE TITLE
specs.ipfs.tech should link to github sources

### ISSUE THREAD

**Issue by lidel** [ipfs/specs#399]
It is not clear how one can get to the sources of the websie, how to open a PR to fix a typo or propose  clarification / discuss potential IPIP.

Every page should have a link to github source of itself.

Some prior art:
- W3C links to commit history in the header preamble:
  > ![2023-04-17_11-12](https://user-images.githubusercontent.com/157609/232439984-0371f069-e227-439b-ae60-433dc814c65d.png)
- Our utility websites have github icon
  > ![2023-04-17_11-10](https://user-images.githubusercontent.com/157609/232440176-749b9cb4-40ba-49f3-b94a-ada313e2d814.png)

I think adding entry after editors to the header of each spec with "Markdown sources" and "Commit history" with GH icon which we already use for personal github lrofile links would be good enough.

@darobin @hacdias thoughts?

**Comment by darobin** [comment]
Yes, absolutely, we have that filed at https://github.com/ipfs/spec-generator/issues/10.

I also like this, which we have in W3C:
<img width="593" alt="image" src="https://user-images.githubusercontent.com/38491/232476924-6be31eb2-1d1a-4583-ac73-8cd90439efcc.png">
